### PR TITLE
[WIP] added files required for builds in app-interface

### PR DESCRIPTION
**Pull request by SteveHNH**
Signed-off-by: Stephen Adams <tsadams@gmail.com>

    ## Secure Coding Practices Checklist GitHub Link
    - https://github.com/RedHatInsights/secure-coding-checklist

    ## Secure Coding Checklist
    - [ ] Input Validation
    - [ ] Output Encoding
    - [ ] Authentication and Password Management
    - [ ] Session Management
    - [ ] Access Control
    - [ ] Cryptographic Practices
    - [ ] Error Handling and Logging
    - [ ] Data Protection
    - [ ] Communication Security
    - [ ] System Configuration
    - [ ] Database Security
    - [ ] File Management
    - [ ] Memory Management
    - [ ] General Coding Practices

### CODE REVIEW

File: unit_test.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Deploy ephemeral DB
+source $CICD_ROOT/deploy_ephemeral_db.sh
+
+# Map env vars set by `deploy_ephemeral_db.sh` if vars the app users are different
+export POSTGRES_ADMIN_PASSWORD=$DATABASE_ADMIN_PASSWORD
+export DB_HOST=$DATABASE_HOST
+export DB_DATABASE=$DATABASE_NAME
+export DB_USERNAME=$DATABASE_USER
+export DB_PASSWORD=$DATABASE_PASSWORD
+export DB_PORT=$DATABASE_PORT
```
Comment (aleccohan) [inline unit_test.sh:14]: Just looked in the config for Remediations, looks like the env variables for db for tests are as follows:
```
DB_USERNAME = postgres
DB_PASSWORD = remediations
DB_DATABASE = remediations
DB_HOST = 127.0.0.1
```

File: unit_test.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $APP_ROOT
+
+# Deploy ephemeral DB
+source $CICD_ROOT/deploy_ephemeral_db.sh
+
+# Map env vars set by `deploy_ephemeral_db.sh` if vars the app users are different
```
Comment (aleccohan) [inline unit_test.sh:8]: Along with the DB will we also be needing to provision a redis instance?

Comment (SteveHNH) [reply unit_test.sh:8]: I don't think your tests require redis as far as I can tell.

File: unit_test.sh (hunk 1 above)
Comment (SteveHNH) [reply unit_test.sh:14]: cool. I think we're good to go here. The ephemeral DB sets its own stuff so we'll just use those to match

**Pull request closed without merging**